smithy-aws-protocol-tests/model/restJson1/http-headers.smithy

```diff
@@ -413,3 +413,57 @@ structure TimestampFormatHeadersIO {
     @httpHeader("X-targetDateTime")
     targetDateTime: DateTime,
 }
+
+/// This example ensures that mediaType strings are base64 encoded in headers.
+@readonly
+@http(uri: "/MediaTypeHeader", method: "GET")
+operation MediaTypeHeader {
+    input: MediaTypeHeaderInput,
+    output: MediaTypeHeaderOutput
+}
+
+apply MediaTypeHeader @httpRequestTests([
+    {
+        id: "MediaTypeHeaderInputBase64",
+        documentation: "Headers that target strings with a mediaType are base64 encoded",
+        protocol: restJson1,
+        method: "GET",
+        uri: "/MediaTypeHeader",
+        headers: {
+            "X-Json": "dHJ1ZQ=="
```

Comment:
Not sure we want to update this since HTTP headers are supposed to be case insensitive, but the headers in both test cases are capital-J Json while the `@httpHeader` strings are lower-j json.

Reply:
Good catch. Fixed.